orderer/common/server/main.go

```diff
@@ -278,6 +278,15 @@ func Main() {
 		conf.General.Authentication.NoExpirationChecks,
 	)
 
+	attestationService := NewAttestationService(
```

Comment:
Let's put it optional for now, and disable it by default?

Reply:
Or we can not attach it for now

Reply:
Updated. The service is now not enabled.  

But, still the dead/unused code still be part of the other future releases of 2.x.  can we move this BFT changes into separate branch.

Reply:
if it's not imported from the main, it is not part of the binary. I don't think we need to put it in a separate branch.